int8 PERCENT_OUTPUT = 0
int8 POSITION = 1
int8 VELOCITY = 2
int8 CURRENT = 3
int8 VOLTAGE = 4
int8 FOLLOWER = 5
int8 MOTION_PROFILE = 6
int8 MOTION_MAGIC = 7
int8 MOTION_PROFILE_ARC = 10
int8 MUSIC_TONE = 13
int8 DISABLED = 15

int8 mode
float64 value